packages/graphiql-plugin-code-exporter/src/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { GraphiQLPlugin } from '@graphiql/react';
+import { useMemo, useRef } from 'react';
+import GraphiQLCodeExporter, {
+  GraphiQLCodeExporterProps,
+} from 'graphiql-code-exporter';
+
+import './graphiql-code-exporter.d.ts';
+import './index.css';
+
+function ExporterPlugin(props: GraphiQLCodeExporterProps) {
+  return <GraphiQLCodeExporter {...props} />;
+}
```

Comment:
That function is just a wrapper around `GraphiQLCodeExporter`, should we remove it and use `GraphiQLCodeExporter` directly in the plugin definition below?